toolkit/src/main/cpp/export/meta_toolkit_export_plugin.cpp

```diff
@@ -176,3 +222,628 @@ PackedStringArray MetaToolkitExportPlugin::_get_android_libraries(const Ref<godo
+			_get_int_option("meta_xr_features/body_tracking", BODY_TRACKING_OPTIONAL_VALUE) == BODY_TRACKING_REQUIRED_VALUE,
+		};
+		r_features.append(body_tracking_info);
+	}
+
+	// Check for hand tracking.
+	bool hand_tracking_enabled = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/hand_tracking");
+	if (hand_tracking_enabled) {
+		r_permissions.append("com.oculus.permission.HAND_TRACKING");
+
+		FeatureInfo hand_tracking_info = {
+			"oculus.software.handtracking",
+			_get_int_option("meta_xr_features/hand_tracking", HAND_TRACKING_OPTIONAL_VALUE) == HAND_TRACKING_REQUIRED_VALUE,
+		};
+		r_features.append(hand_tracking_info);
+
+		MetadataInfo hand_tracking_version_metadata = {
+			"com.oculus.handtracking.version",
+			"V2.0"
+		};
+		r_metadata.append(hand_tracking_version_metadata);
+
+		int hand_tracking_frequency = _get_int_option("meta_xr_features/hand_tracking_frequency", HAND_TRACKING_FREQUENCY_LOW_VALUE);
+		const String hand_tracking_frequency_label = (hand_tracking_frequency == HAND_TRACKING_FREQUENCY_LOW_VALUE) ? "LOW" : "HIGH";
+		MetadataInfo hand_tracking_frequency_metadata = {
+			"com.oculus.handtracking.frequency",
+			hand_tracking_frequency_label
+		};
+		r_metadata.append(hand_tracking_frequency_metadata);
+	}
+
+	// Check for passthrough.
+	bool passthrough_enabled = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/meta/passthrough");
+	if (passthrough_enabled) {
+		FeatureInfo passthrough_info = {
+			"com.oculus.feature.PASSTHROUGH",
+			_get_int_option("meta_xr_features/passthrough", PASSTHROUGH_OPTIONAL_VALUE) == PASSTHROUGH_REQUIRED_VALUE,
+		};
+		r_features.append(passthrough_info);
+	}
+
+	// Check for render model.
+	bool render_model_enabled = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/meta/render_model");
+	if (render_model_enabled) {
+		r_permissions.append("com.oculus.permission.RENDER_MODEL");
+
+		FeatureInfo render_model_info = {
+			"com.oculus.feature.RENDER_MODEL",
+			_get_int_option("meta_xr_features/render_model", RENDER_MODEL_OPTIONAL_VALUE) == RENDER_MODEL_REQUIRED_VALUE,
+		};
+		r_features.append(render_model_info);
+	}
+
+	// Check for anchor api.
+	bool use_anchor_api = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/meta/anchor_api");
+	if (use_anchor_api) {
+		r_permissions.append("com.oculus.permission.USE_ANCHOR_API");
+	}
+
+	// Check for anchor sharing.
+	bool use_anchor_sharing = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/meta/anchor_sharing");
+	if (use_anchor_sharing) {
+		r_permissions.append("com.oculus.permission.IMPORT_EXPORT_IOT_MAP_DATA");
+	}
+
+	// Check for scene api.
+	bool use_scene_api = ProjectSettings::get_singleton()->get_setting_with_override("xr/openxr/extensions/meta/scene_api");
+	if (use_scene_api) {
+		r_permissions.append("com.oculus.permission.USE_SCENE");
+	}
```

Comment:
This should also check for environment depth as done in https://github.com/GodotVR/godot_openxr_vendors/blob/3b4fb43d4e88be334f0ce73244490216861f46fd/plugin/src/main/cpp/export/meta_export_plugin.cpp#L418